course.py

```diff
@@ -249,3 +249,45 @@ def add_duplicate_course():
         db.session.rollback()
 
     return redirect(url_for('course.course_info', course_id=course_id, current_year=year))
+
+
+@course_bp.route('/evaluations/<int:user_id>/<int:current_year>')
+@login_required
+def evaluations(user_id, current_year):
+    courses = db.session.query(Course).filter_by(year=current_year).all()
```

Comment:
Maybe the courses that were already evaluated can be filtered out, or, their evaluation can be replaced by a newer one.